File: Hospital/HospitalClassLib/Model/Feedback.cs
using Hospital.Model.Enumerations;
using System;

namespace Hospital.Model
{
    public class Feedback
    {
        public int Id { get; set; }
        public int PatientId { get; set; }
        public Patient Patient { get; set; }

        public FeedbackStatus FeedbackStatus { get; set; }  

        public DateTime CreatedDate { get; set; }
        public string Text { get; set; }
        public bool IsPublishable { get; set; }
    }
}

File: WebClients/ManagerWebClient/src/app/interfaces/feedback.ts
import { IPatient } from "./patient";


export enum FeedbackStatus {
    Pending, 
    Rejected, 
    Approved
  }

export interface IFeedback {
    id: number;
    patientId: number;
    patient: IPatient;
    feedbackStatus: FeedbackStatus;
    createdDate: Date;
    text: string;
    isPublishable: boolean;
        
}

